### ISSUE TITLE
What is "Sakura.exe" and "helplerSakura.exe"?

### ISSUE THREAD

**Issue by lugiah** [SoulGirlJP/AzureV316#16]
Hi. Thanks for your sharing and guide for building.
There were couple of alerts (from Window Defender) in the client.
There are two suspicious program which are "Sakura.exe" and "helperSakura.exe" in the client.
Below is what Window Defender detected
   Trojan:Win32/Tiggre!rfn from file: D:\AzureMSV316\AzureLocal\helperSakura.exe
   Trojan:Win32/Casur.A!cl from file: D:\AzureMSV316\AzureSetup316.zip->AzureLocal/Sakura.exe

I understand that most of alerts are false-positive when you build private server, but just want to know what program really is and what for. Thank you

**Comment by KOOKIIEStudios** [comment]
Thank you! It means a lot to me and I'm glad you've found the Wiki helpful.

First of all, I should warn you that SoulGirlJP has deprecated this project about a year or so ago. If you'd like to try your luck, perhaps you can try reaching SoulGirlJP on Discord directly, or try some of the forks like [Electron](https://github.com/Bratah123/ElectronMS) instead.

I'm not sure of the client you're using, but the oldest/original clients did not seem to have these files - my own copy does not have them. Since the Google Drive link will die every so often, SoulGirlJP has re-uploaded and refreshed the link several times, and I can not say for certain that it's the exact same every time (because I've never checked). What mine does have, though is a similarly named `SakuraShare.exe`, whose `ini` file seems to contain data pertaining to WZ file versioning and some kind of key for dealing with the server. I haven't decompiled the executable to check for safety, but will note that AFAIK, that client is not actually part of the project and was sourced from an external party, and as such would not be supported by the project team anyway. If you're uncomfortable with not knowing the contents of the client, you could try making your own using a clean v316 client, and [ElectronClient](https://github.com/Bratah123/ElectronClient) (more details in the [Electron Discord](https://discord.com/channels/793738824547041296/793738943207833640/890471314542952458)). It's quick and dirty (and some MSPS devs routinely scoff at it) but the source code is open for you to read and check.

**TL;DR**: *No one knows cuz it wasn't made by us. There are [alternatives](https://github.com/Bratah123/ElectronClient) for the client. Also, dead project. If you want to try your luck with getting support, DM the owner or try a [fork](https://github.com/Bratah123/ElectronMS) instead.*

- *The owner's new Discord handle post-apocalypse (Discord mass-banning MSPS devs): `SoulsJP#3857`*
- *The owner's development studio: [Indoors Studio](https://discord.gg/wJrbW8CK)*
